lib/streamlit/elements/deck_gl_json_chart.py

```diff
@@ -156,12 +157,18 @@ def marshall(
 ) -> None:
     if pydeck_obj is None:
         spec = json.dumps(EMPTY_MAP)
+        id = ""
     else:
         spec = pydeck_obj.to_json()
+        json_string = json.dumps(pydeck_obj.to_json())
```

Comment:
Why is the `json.dumps` needed here? isn't `pydeck_obj.to_json` already converting this to a json string?